3.5/Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM microsoft/iis
+FROM microsoft/dotnet-framework:3.5
 
-ADD install /build
-RUN DISM /Online /Add-Package /PackagePath:C:\build\microsoft-windows-netfx3-ondemand-package.cab & \
-    DISM /Online /Add-Package /PackagePath:C:\build\patch\Windows10.0-KB3213986-x64.cab  & \
-    rd /s /q C:\build & \
+RUN powershell -Command Add-WindowsFeature Web-Server & \
     powershell -Command Add-WindowsFeature Web-Asp-Net & \
     %windir%\SysWOW64\inetsrv\appcmd set app "Default Web Site/" /applicationPool:".NET v2.0" & \
```

Comment:
Instead of changing the AppPool, we should changed .NET Framework version of the DefaultAppPool to .NET 2.0 to work with the new ServiceMonitor.exe
See https://github.com/Microsoft/iis-docker/pull/41